src/ngraph/ops.hpp

```diff
@@ -162,6 +162,7 @@
 #include "ngraph/op/scatter_add.hpp"
 #include "ngraph/op/scatter_elements_update.hpp"
 #include "ngraph/op/scatter_nd_add.hpp"
+#include "ngraph/op/scatter_nd_update.hpp"
```

Comment:
There is a scatter_nd_add class already which is an op with the same behaviour except the reduction function (add vs update). We should create a base class for all ScatterND operations like we did for the Scatted operations (ngraph/src/ngraph/op/util/scatter_base.hpp) and use it for both ScatterNDAdd and ScatterNDUpdate.

Reply:
can we address this in another PR?

Reply:
I don't like this idea. We should implement the code as good as possible from the very beginning.